src/parsing/lpParser.mly

```diff
@@ -242,7 +250,13 @@ proof_end:
   | ADMITTED { make_pos $sloc Syntax.P_proof_admitted }
   | END { make_pos $sloc Syntax.P_proof_end }
 
-proof: BEGIN ts=terminated(tactic, SEMICOLON)* pe=proof_end { ts, pe }
+bracket_tactic :
+  L_CU_BRACKET tree_list=tree* R_CU_BRACKET { Tactic(None, tree_list) }
+
+tree :
```

Comment:
tree is too generic for a name. Thanks to use tactic_tree for instance instead. (Note that we already have another notion of tree in LP, namely decision trees for matching.)